TestModels/SimpleTypes/SimpleTimestamp/test/SimpleTimestampImplTest.dfy

```diff
@@ -19,7 +19,7 @@ module SimpleTimestampImplTest {
       modifies client.Modifies
       ensures client.ValidState()
     {
-        var dafnyTimestamp := "2024-06-11T12:34:56";
+        var dafnyTimestamp := "2024-06-11T12:34:56Z";
```

Comment:
Just wondering, what version of python does the customer have to be in because until python 3.10, it does not support military time stamp (timestamp with Z on the end)

Reply:
3.11. This constraint comes from Smithy-Python

Reply:
I'll also note that I'm not even running SimpleTimestamp in Python, since I'm claiming that timestamps aren't supported for local services.